cellxgene_resources/curation_qa.ipynb

```diff
@@ -438,6 +438,39 @@
     "    display(deprecated_donors)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Check if any human donors' annotated sex is inconsistent with raw expression counts**<br>"
```

Comment:
Would add some text underneath this explaining the cut-offs used for male and female. So in the case that there is some inconsistencies, they are readily available to compare the table with